src/components/SearchSection.test.tsx

```diff
@@ -159,9 +159,12 @@ describe("SearchSection", () => {
     fireEvent.click(sectorDropdown);
     expect(screen.getByText("Power")).toBeInTheDocument();
 
-    const categoryDropdown = screen.getByText("Category");
-    fireEvent.click(categoryDropdown);
-    expect(screen.getByText("Policy")).toBeInTheDocument();
-    expect(screen.getByText("Forecast")).toBeInTheDocument();
+    const pathwayDropdown = screen.getByText("Pathway Type");
+    fireEvent.click(pathwayDropdown);
+    //TODO: Re-enable these tests
+    //expect(screen.getByText("Exploration")).toBeInTheDocument();
+    //expect(screen.getByText("Normative")).toBeInTheDocument();
+    //expect(screen.getByText("Policy")).toBeInTheDocument();
+    //expect(screen.getByText("Projection")).toBeInTheDocument();
```

Comment:
We should tackle all TODOs before merging to `main`. (Or remove them and open an issue if they really have nothing to do with this PR... though it seems that they are related).

Reply:
Note that the dropdown only loads what's in the data (no options that yield no results). Add one of each type to the testing data

Reply:
Sure. But my request is that we don't get in the habit of merging TODOs into `main`. 
It's fine if we want to kick that can down the road, but it should be tracked in an issue, not in the codebase